### ISSUE TITLE
Passing incorrect animationName

### ISSUE THREAD

**Issue by Bayramito** [rive-app/rive-react-native#259]
Hi,

This is a very common issue, i dunno if i'm the only one who gets it but,

once you pass the `animationName` to the ref methods, the app is crashing.

In the `.riv` files sometimes the animation names can be uppercase or camel case, so we need some sort of fix to handle this situation. Like checking or converting both requested and existing animations names to lowercase.

**Comment by CoopCodes** [comment]
Yeah I get the same, if I get the state machine name, or the animation name or input name wrong it just crashes without any error. Would be good if it just didn't do anything, or just gave a warning.

**Comment by guval-gh** [comment]
Same here.

And even if the error is normally handled like in this example:  https://github.com/rive-app/rive-react-native/blob/ba12e7a5117a4d948af66f10dcd9d2ff304d2204/example/src/ErrorHandledManually.tsx

The `url` is `IncorrectRiveFileUrl` or if `resourceName` is `MalformedFile`, app instant crashes on iOS for me (didn't test on Android).
